CONTRIBUTING.md

```diff
@@ -28,6 +28,23 @@ Here are a few things you can do that will increase the likelihood of your pull
 * Keep your change as focused as possible. If there are multiple changes you would like to make that are not dependent upon each other, consider submitting them as separate pull requests.
 * [Write a good commit message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
 
+### Unity & LTS support
+
+As we are a group of volunteers, we can only officially support the versions of Unity that are actively supported by Unity themselves. Please check the official [LTS FAQ](https://support.unity.com/hc/en-us/articles/4403332003348-What-is-a-Unity-LTS-Long-Term-Support-version-and-what-can-I-expect-from-it-) for more up-to-date information.
```

Comment:
I mean, I know yall have the patreon but I can't imagine you're able to provide legacy support like Oracle lol

(though that might be an idea...)

Reply:
Yeah in an ideal world we'd support everyone and everything but that is not gonna ever happen.